client/src/components/Header/index.jsx

```diff
@@ -0,0 +1,33 @@
+'use client';
+import React from "react";
+import styles from "./styles.module.scss";
+import Image from "next/image";
+
+import logoLight from "@assets/logo-light.svg";
+
+const Header = () => {
+	return (
+		<div className={styles.header}>
+            <div className={styles.heading}>
+                <div className={styles.logo}>
+                    <Image
+                        src={logoLight}
+                        width={150}
+                        height={30}
+                        alt="icon"
+                    />
```

Comment:
The logo also exists already here https://github.com/Erikang20/Kanban-task-management-web-app/blob/main/client/src/components/Header/logo.tsx

Reply:
It wasn't available at the time I pulled and started working on the task. But you can replace it if that's okay.

Reply:
yeah this is probably my fault. These are some changes I made that I mentioned yesterday that I needed to complete my last task.  What I'd do is to rebase against main and run the app and see what elements are not longer needed  ;) Let me know if you need help with rebase.